README.md

```diff
@@ -25,77 +35,84 @@ Add this dependency to your project's POM:
 
+### Basic Usage
 
-An example application which demonstrates both Basic and Advanced usage is at [uid2-examples](https://github.com/UnifiedID2/uid2-examples/tree/main/publisher/uid2-java-test-site).
+If you're using the SDK's HTTP implementation, follow these steps.
 
-### Basic usage
 1. Create an instance of PublisherUid2Client as an instance variable.
  
    `private final PublisherUid2Client publisherUid2Client = new PublisherUid2Client(UID2_BASE_URL, UID2_API_KEY, UID2_SECRET_KEY);`
 
-2. When a user authenticates and authorizes the creation of a UID2:
+2. When a user authenticates and authorizes the creation of a UID2: **(gwh/JN_02 action? Does the user have to do something here?)**
  
    `IdentityTokens identity = publisherUid2Client.generateToken(TokenGenerateInput.fromEmail(emailAddress));`
  
-
 #### Standard Integration
-If you're using [standard (client and server) integration](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/guides/publisher-client-side.md):
+
+If you're using [standard (client and server) integration](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/guides/publisher-client-side.md), follow these steps. **(gwh/JN_03 definitely only one step?)**
+
 1. Send this identity as a JSON string back to the client (to use in the [identity field](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md#initopts-object-void)) using: `identity.getJsonString()`
 
 #### Server-Only Integration
+
 If you're using [server-only integration](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/guides/custom-publisher-integration.md):
 
 1. Store this identity as a JSON string in the user's session, using: `identity.getJsonString()`
 2. To use the user's UID2 token, use `identity.getAdvertisingToken()`
 
 3. When the user accesses another page, or on a timer, determine whether a refresh is needed:
-   1. Retrieve the identity JSON string from the user's session, then call:
+   1. Retrieve the identity JSON string from the user's session, then call: **(gwh/JN_04 should this be two steps?)**
 
       `IdentityTokens identity = IdentityTokens.fromJsonString(identityJsonString);`
-   2. Determine if the identity is refreshable (ie, the refresh token hasn't expired):
+   2. Determine if the identity is refreshable (that is, the refresh token hasn't expired):
 
-      ` if (identity == null || !identity.isRefreshable()) { we must no longer use this identity (eg, remove this identity from the user's session) }`
+      ` if (identity == null || !identity.isRefreshable()) { we must no longer use this identity (for example, remove this identity from the user's session) }`
    3. Determine if a refresh is needed:
 
       `if (identity.isDueForRefresh()) {..}`
-4. If a refresh is needed, call:
+4. If a refresh is needed, call: **(gwh/JN_05 can we state what this is doing? Initiate token refresh?)**
```

Comment:
This is step 3 here https://github.com/IABTechLab/uid2docs/blob/main/api/v2/guides/custom-publisher-integration.md

Reply:
Reworded. Thx.